eng/scripts/Language-Settings.ps1

```diff
@@ -6,26 +6,63 @@ $packagePattern = "*.tgz"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=javascript%2F&delimiter=%2F"
 
-function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
+if ((Get-Command npm | Measure-Object).Count -eq 0 ) 
 {
-  $projectPath = Join-Path $pkgPath "package.json"
-  if (Test-Path $projectPath)
+  LogError "Could not locate npm. Install NodeJS (includes npm and npx) https://nodejs.org/en/download"
+  exit 1
+}
+
+function Get-javascript-PackageInfoFromRepo
+{
+  [CmdletBinding()]
+  Param(
```

Comment:
Lets not change these parameters until we change the common code.

Reply:
Hmm,
I thought the idea was to set this up such that I can pass positional parameters  on the `eng/common` side later on and have it still work. Then circle back and clean up the other parameters.